invenio_app_ils/patrons/anonymization.py

```diff
@@ -180,22 +180,28 @@ def delete_user_account(patron_pid):
 
     with db.session.begin_nested():
         d = db.session.query(userrole).filter(userrole.c.user_id == patron_pid)
-        dropped += d.delete(synchronize_session=False)
+        dropped += d.delete(synchronize_session=False) or 0
 
         dropped += SessionActivity.query.filter(
             SessionActivity.user_id == patron_pid
-        ).delete()
+        ).delete() or 0
 
         dropped += UserIdentity.query.filter(
             UserIdentity.id_user == patron_pid
-        ).delete()
-        dropped += RemoteAccount.query.filter(
+        ).delete() or 0
+
+        ra = RemoteAccount.query.filter(
             RemoteAccount.user_id == patron_pid
-        ).delete()
+        ).one_or_none()
+        if ra:
+            dropped += RemoteToken.query.filter(
```

Comment:
@kprzerwa @FlorianCassayre fixed Sentry exception for RemoteToken

Reply:
Great, thanks :+1:

Reply:
👍